var/spack/repos/builtin/packages/py-pennylane-lightning/package.py

```diff
@@ -3,7 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-
+import spack.build_systems.cmake
```

Comment:
Here `spack.build_systems.cmake` is imported, but we are still using `spack.build_systems.cmake.CMakeBuilder` at [line 62](bbc7c0a9fcb60b7f6a12fb5e0636d625ec1c3e60b212cc65ed952492abe9eea3R62). Was the goal to use a more targeted import as in `var/spack/repos/builtin/packages/py-pennylane-lightning-kokkos/package.py`? Should we make that change in other files like GROMACS'?